<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-SDLQ3M27" failures="2" tests="2" name="PersonalLoan" time="10.265" errors="0" timestamp="2021-01-27T20:55:18 IST">
  <testcase classname="COM.SELENIUM.TOPICS._23_SoftAssert" name="TestCase_1" time="10.201">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Ho] but found [Home]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Ho] but found [Home]
at COM.SELENIUM.TOPICS._23_SoftAssert.TestCase_1(_23_SoftAssert.java:44)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TestCase_1 -->
  <testcase classname="COM.SELENIUM.TOPICS._23_SoftAssert" name="TestCase_1" time="10.233">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;org.openqa.selenium.TakesScreenshot.getScreenshotAs(org.openqa.selenium.OutputType)&amp;quot; because &amp;quot;ts&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.TakesScreenshot.getScreenshotAs(org.openqa.selenium.OutputType)" because "ts" is null
at COM.SELENIUM.TOPICS._42_CustomListener.ScreenShot(_42_CustomListener.java:34)
at COM.SELENIUM.TOPICS._42_CustomListener.onTestFailure(_42_CustomListener.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- TestCase_1 -->
</testsuite> <!-- PersonalLoan -->
